### DISCUSSION TITLE
using Foreground-aware Relation Network (FarSeg)

### DISCUSSION THREAD

**Discussion by kethan52** [torchgeo/torchgeo#600] [General]
Hi, can i know how to use the Foreground-aware Relation Network (FarSeg) model for the segmentation task or can I know how can we train for the custom dataset using it

**Comment by adamjstewart** [comment]
As far as I know, training a FarSeg model is no different from training a normal ResNet. If you're new to PyTorch, I would recommend checking out some of their tutorials, like [Quickstart](https://pytorch.org/tutorials/beginner/basics/quickstart_tutorial.html) or [Optimization](https://pytorch.org/tutorials/beginner/basics/optimization_tutorial.html).

**Comment by adamjstewart** [comment]
I'm assuming that the above links answer your question, but let me know if there's anything else I can do to help!
